### ISSUE TITLE
Attempting to Update Languages (both Windows and OSX)

### ISSUE THREAD

**Issue by door43app** [unfoldingWord-dev/ts-desktop#733]
Notes
======
Attempting to Update Languages

Environment
======
Environment Key | Value
:--: | :--:
Version |9.1.0
Operating System | Darwin
Platform | darwin
Release | 15.5.0
Architecture | x64

Log History
======
```javascript
2016-07-07 16:5:48 I/migrator.js:31:2: Migrated projects
	[]
2016-07-07 16:10:15 E/ts-dashboard.html:320:3: Update Failed
	Could not update language list

```

**Comment by cckozie** [comment]
Renamed translationStudio folder to get a fresh start. Tested logging in multiple ways. Then did an Update Languages and got this message.
![image](https://cloud.githubusercontent.com/assets/8171759/16669942/9968c838-445d-11e6-878d-105b7e2ac9ec.png)
When I clicked Update Languages again, the app crashed.
![image](https://cloud.githubusercontent.com/assets/8171759/16670026/f1e75998-445d-11e6-904f-6202fd692d41.png)
